lib/phlex/rails/sgml.rb

```diff
@@ -18,27 +18,34 @@ def helpers
 					end
 				end
 
-				def render(*args, **, &block)
+				def render(*args, **kwargs, &block)
 					renderable = args[0]
+					partial = kwargs.delete(:partial)
```

Comment:
Should this move down to the `when nil` clause?

Reply:
I just caught that myself. Pushed up a change.